include/main_functions_batch.h

```diff
@@ -38,6 +38,6 @@ char* edit_batch(char* line, dict_t* dict, int verbosity, int lnum);
  *
  * return: char** array of lines, to be printed if quiet mode
  */
-char** batch_mode(char* filename, dict_t* dict, bool* quit, int verbosity);
+char **batch_mode(char *filename, dict_t *dict, bool *quit, int verbosity);
```

Comment:
are we not changing this to a linked list return type?

Reply:
No, see your pull request. Mark commented on it on this.